train_hierarchialXC.py

```diff
@@ -41,6 +46,25 @@
 
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ Train your model ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-my_hierarchical_XC.train(loader, multiclass.OneVsRestClassifier, estimator=svm.SVC())
+my_hierarchical_XC.train(loader, multiclass.OneVsRestClassifier, estimator=svm.SVC(gamma="scale"))
 predictions = my_hierarchical_XC.predict(loader.get_data()[0])
-print(predictions.toarray())
+
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ Calculate the metrics ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+pred_y = predictions.toarray()
+actual_y = []
+for x, y in iter(loader):
+    actual_y.append(y.numpy().reshape(-1))
```

Comment:
loop is redundant. Make this: `actual_y = loader.get_classes().toarray()` or sth like this.